pages/guide.md

```diff
@@ -18,6 +18,7 @@ This page provides implementation guidance for agencies by the White House Offic
   * [Do I need to shut off port 80?](#do-i-need-to-shut-off-port-80%3f)
   * [What does "all Federal agency domains or subdomains" include?](#what-does-"all-federal-agency-domains-or-subdomains"-include%3f)
   * [What about domains that are only used to redirect visitors to other websites?](#what-about-domains-that-are-only-used-to-redirect-visitors-to-other-websites%3f)
+  * [Do domains that redirect to other external domains need to redirect internally to HTTPS before redirecting externally?](#do-domains-that-redirect-to-other-external-domains-need-to-redirect-internally-to-https-before-redirecting-externally%3f)
```

Comment:
Each FAQ answer also gets a link in the ToC at the top of the page for easy navigation.